docs/pipelines.md

```diff
@@ -0,0 +1,24 @@
+# ARO HCP Pipelines
```

Comment:
This is destined to be outdated soon. I think we should generate this info from the Pipeline files instead. ADO Pipelines information could be part of some mapping file. (or we can get a token to retrieve this info?)

Reply:
good point. i've created a ticket to track this
https://issues.redhat.com/browse/ARO-15637